admin_app/src/app/content/edit/page.tsx

```diff
@@ -224,6 +240,31 @@ const ContentBox = ({
             placeholder="Add Tags"
           />
         )}
+        filterOptions={(options, params) => {
+          const filtered = filter(options, params);
+
+          const { inputValue } = params;
+          const isExisting = options.some(
+            (option) => inputValue === option.tag_name
+          );
+          if (inputValue !== "" && !isExisting) {
+            filtered.push({ tag_id: 0, tag_name: `Add "${inputValue}"` });
+          }
+
+          return filtered;
+        }}
+        renderOption={(props, option) => {
+          if (option.tag_name) {
+            return (
+              <li {...props}>
+                <Button fullWidth onClick={() => handleNewTag(option.tag_name)}>
+                  {option.tag_name}
+                </Button>
+              </li>
```

Comment:
What do you think of this? Seems cleaner and gets rid of the weird button-inside-a-list effect

```suggestion
              <li {...props} onClick={() => handleNewTag(option.tag_name)}>
                  {option.tag_name}
              </li>
```